main.tf

```diff
@@ -1 +1,609 @@
-# Your Terraform code goes here :)
+
+data "aws_region" "current" {}
+
+// Todo: replace with iam role module
+resource "random_string" "external_id" {
+  length           = 256
```

Comment:
I see in CloudFormation we set this to a 7-character value.

I think larger is better but I'd like to stay consistent. Do we set the length to 256 in other terraform modules too?

Reply:
We set this as a variable in other modules, and default it to 16 characters: https://github.com/lacework/terraform-aws-iam-role/blob/main/variables.tf#L13-L17